### DISCUSSION TITLE
portrait mode

### DISCUSSION THREAD

**Discussion by saskiagro** [mediacms-io/mediacms#528] [Feature Requests]
hi. I´m working with videos in portrait mode.
How can i modfiy the thumbnail output and view to portrait instead of landscape mode? 

Thank you.

**Comment by mgogoulos** [comment]
> hi. I´m working with videos in portrait mode. How can i modfiy the thumbnail output and view to portrait instead of landscape mode?
> 
> Thank you.

Hi, I don't think that is possible now, it would require work on the templates to support it.
